### PR TITLE
feat: Card Component added

### PR DESCRIPTION
**Pull request by Sen-442b**
**Features:-** 

-  Horizontal card
- Card with badges
- Card with shadows
- Text only card

**Additional Features** 
- Created Global CSS file and imported all the local files inside it 
- Reset a few elements default style

**Issues**
- Performance issues while loading the css
- Poor Design

### CODE REVIEW

**Comment by Sen-442b** [comment]
image
image

File: Card/source/card.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
+@import url("../../global.css");
+
+
+
+
+
+
+
+
+
```
Comment (manojasarsa) [inline Card/source/card.css:10]: You can remove the extra spaces.

File: Card/source/card.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    font-weight: 500;
+    letter-spacing: 0.0125em;
+    margin-bottom: 0;
+  }
+
+
+
+  .image-child{
+      /* max-width: 100%;
+      max-height: 100%; */
```
Comment (manojasarsa) [inline Card/source/card.css:104]: Also, the comments can be removed.

**Review by manojasarsa — APPROVED** [review]
Suggestion : You can add a class for "display: flex, flex-direction, justify-content" and use that everytime needed.  
Rest all looks good! 👍

File: Alert-Button/source/button.css [hunk 3]
```diff
@@ -1,10 +1,11 @@
+@import url("../../global.css");
 body {
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
     Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
 }
 
 .btn {
-  all: unset;
+
   outline: none;
```
Comment (nshanbhag20001412) [inline Alert-Button/source/button.css:10]: you can remove such space

File: Card/index.html [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+     
+    </div>
+
+   
+
+
+</body>
+</html>
+
+<!--
```
Comment (nshanbhag20001412) [inline Card/index.html:181]: code which are not required can be removed

File: Card/source/card.css [hunk 5]
```diff
@@ -0,0 +1,254 @@
+@import url("../../global.css");
+
```
Comment (nshanbhag20001412) [inline Card/source/card.css:2]: space

File: Card/source/card.css [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+    flex-direction: column;
+    box-shadow: var(--universal-box-shadow);
+ 
+    
+
+  }
+
+   .image-container {
+    position:relative;
+
```
Comment (nshanbhag20001412) [inline Card/source/card.css:49]: same space

**Review by nshanbhag20001412 — APPROVED** [review]
apart from the suggestions below rest looks good to me

File: Card/source/card.css [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+      box-shadow: var(--universal-box-shadow);
+  }
+  /* test overflow elipses */
+  
+  .headline-typography {
+    
+    font-size: 1.25rem;
+    font-weight: 500;
+    letter-spacing: 0.0125em;
+    margin-bottom: 0;
```
Comment (BelideNikhil) [inline Card/source/card.css:97]: use variables for font weight and size as well if possible
Rest all is good work!

**Review by BelideNikhil — APPROVED** [review]

**Review by bipin7yadav — APPROVED** [review]

**Pull request merged**